### ISSUE TITLE
Allow inclusion of local rules

### ISSUE THREAD

**Issue by 19D** [hadolint/hadolint#646]
- [ ] This is a bug report
- [ x ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior
`hadolint --add-rule path/to/rule.hs Dockerfile` allows the inclusion of a local, pipeline specific rule file to be included in the linting process
### Actual behavior
Using the ironbank hadolint image prevents the addition of extra rules into the source code.
### Steps to reproduce the behavior

```yaml
lint_dockerfile:
  image: hadolint/hadolint:latest-debian
  # image: ghcr.io/hadolint/hadolint:latest-debian
  script:
    - hadolint Dockerfile
```

**Additional environment details (OS, stack version, etc.)**
Gitlab CI

**Comment by lorenzo** [comment]
it is a cool idea and something I have wanted myself in the past, but the complexity of implementing this both for hadolint and for people writing their own rules would be too high.
My suggestion is to contribute you own rules to hadolint and we can help you make them generic enough that they can be used by others.
